src/cpu/cpu_primitive.hpp

```diff
@@ -96,6 +96,16 @@
     if (zero_points_ptr == nullptr) return status::invalid_arguments; \
     MAYBE_UNUSED(zero_points_ptr);
 
+#define DEFINE_ZERO_POINTS_BUFFER_F32(zero_points_ptr, mem_arg) \
```

Comment:
I'm just wondering will the following version be easier to maintain, when code in oneapi-src/oneDNN changes.
```
#define DEFINE_ZERO_POINTS_BUFFER_TYPE(zero_points_ptr, mem_arg, type) \
        type CONCAT2(default_zero_point_, mem_arg) = 0; \
        ...
#define DEFINE_ZERO_POINTS_BUFFER(zero_points_ptr, mem_arg) \
        DEFINE_ZERO_POINTS_BUFFER_TYPE(zero_points_ptr, mem_arg, int32_t) \
#define DEFINE_ZERO_POINTS_BUFFER_F32(zero_points_ptr, mem_arg) \ 
        DEFINE_ZERO_POINTS_BUFFER_TYPE(zero_points_ptr, mem_arg, float) \
```

Reply:
Agree, changed